app/src/main/java/com/thoughtworks/aceleradora/solicitacao/dominio/Solicitacao.java

```diff
@@ -39,7 +49,7 @@
     public Solicitacao() {
     }
 
-    public Solicitacao(String nome, String genero, String situacao, String telefone, String cadeirante, float peso, LocalDate dataNascimento, Endereco endereco, List<Acompanhante> acompanhantes) {
+    public Solicitacao(String nome, String genero, String situacao, String telefone, String cadeirante, float peso, LocalDate dataNascimento, LocalDate dataEntrada, LocalDate dataSaida, LocalDate dataTransplante, Endereco endereco, List<Acompanhante> acompanhantes) {
```

Comment:
```suggestion
    public Solicitacao(String nome, 
                                   String genero, 
                                   String situacao, 
                                   String telefone, 
                                   String cadeirante, 
                                   float peso, 
                                   LocalDate dataNascimento, 
                                   LocalDate dataEntrada,
                                   LocalDate dataSaida, 
                                   LocalDate dataTransplante, 
                                   Endereco endereco, 
                                   List<Acompanhante> acompanhantes) {
```

O que acham? Me parece ficar mais legível.

Reply:
Parece bom.